### PR TITLE
Drop literal_block comments and fix a few code blocks

### PR DESCRIPTION
**Pull request by guardrex**
* This gets all of the literal_block comment lines out, except for a few that I took out in the MVC Views overview PR (pending review) ... that PR takes care of them in that topic.
* I also touch a few code block lang idents and spacing as I passed thru.

### CODE REVIEW

File: aspnet/webhooks/index.md [hunk 1]
```diff
@@ -34,9 +34,7 @@ WebHooks is a pattern which means that it varies how it is used from service to
 
 Typically the HTTP POST request contains a JSON object or HTML form data determined by the WebHook sender including information about the event causing the WebHook to trigger. For example, an example of a WebHook POST request body from [GitHub](http://www.github.com/) looks like this as a result of a new issue being opened in a particular repository:
 
-<!-- literal_block {"names": [], "classes": [], "dupnames": [], "xml:space": "preserve", "backrefs": [], "ids": []} -->
-
-````javascript
+```javascript
```
Comment (scottaddie) [inline aspnet/webhooks/index.md]: Change this to json

File: aspnetcore/migration/identity.md [hunk 2]
```diff
@@ -105,8 +101,6 @@ With identity services configured for the application and data access configured
 
 Update _Layout.cshtml; uncomment the @Html.Partial line:
 
-<!-- literal_block {"ids": [], "names": [], "highlight_args": {}, "backrefs": [], "dupnames": [], "linenos": false, "classes": [], "xml:space": "preserve", "language": "none"} -->
-
 ```none
```
Comment (scottaddie) [inline aspnetcore/migration/identity.md]: Change to cshtml

File: aspnetcore/migration/identity.md [hunk 3]
```diff
@@ -119,8 +113,6 @@ Now, add a new MVC View Page called _LoginPartial to the Views/Shared folder:
 
 Update _LoginPartial.cshtml with the following code (replace all of its contents):
 
-<!-- literal_block {"ids": [], "names": [], "highlight_args": {}, "backrefs": [], "dupnames": [], "linenos": false, "classes": [], "xml:space": "preserve", "language": "c#"} -->
-
 ```csharp
```
Comment (scottaddie) [inline aspnetcore/migration/identity.md]: Change to cshtml

File: aspnetcore/mvc/views/razor.md [hunk 4]
```diff
@@ -39,13 +39,13 @@ HTML containing `@` symbols may need to be escaped with a second `@` symbol. For
 
 ```html
```
Comment (scottaddie) [inline aspnetcore/mvc/views/razor.md]: Change to cshtml

File: aspnetcore/mvc/views/razor.md [hunk 5]
```diff
@@ -39,13 +39,13 @@ HTML containing `@` symbols may need to be escaped with a second `@` symbol. For
...
 <p>@@Username</p>
-   ```
+```
 
 would render the following HTML:
 
 ```html
```
Comment (scottaddie) [inline aspnetcore/mvc/views/razor.md]: Change to cshtml

File: aspnetcore/mvc/views/razor.md [hunk 6]
```diff
@@ -110,13 +108,13 @@ C# expressions that evaluate to a string are HTML encoded. C# expressions that e
...
```
Comment (scottaddie) [inline aspnetcore/mvc/views/razor.md]: Change to cshtml

File: aspnetcore/mvc/views/razor.md [hunk 7]
```diff
@@ -131,13 +129,13 @@ The following Razor markup:
...
```
Comment (scottaddie) [inline aspnetcore/mvc/views/razor.md]: Change to cshtml

File: aspnetcore/mvc/views/razor.md [hunk 8]
```diff
@@ -426,43 +420,43 @@ The `@using` directive will add the c# `using` directive to the generated razor
...
 
-```none
+```cshtml
 @model TypeNameOfModel
-   ```
+```
 
 For example, if you create an ASP.NET Core MVC app with individual user accounts, the *Views/Account/Login.cshtml* Razor view contains the following model declaration:
 
 ```csharp
```
Comment (scottaddie) [inline aspnetcore/mvc/views/razor.md]: Change to cshtml

File: aspnetcore/mvc/views/razor.md [hunk 9]
```diff
@@ -426,43 +420,43 @@ The `@using` directive will add the c# `using` directive to the generated razor
...
 @model LoginViewModel
-   ```
+```
 
 In the preceding class example, the class generated inherits from `RazorPage<dynamic>`. By adding an `@model` you control what’s inherited. For example
 
 ```csharp
```
Comment (scottaddie) [inline aspnetcore/mvc/views/razor.md]: Change to cshtml

File: aspnetcore/mvc/views/razor.md [hunk 10]
```diff
@@ -426,43 +420,43 @@ The `@using` directive will add the c# `using` directive to the generated razor
...
 Generates the following class
 
 ```csharp
 public class _Views_Account_Login_cshtml : RazorPage<LoginViewModel>
-   ```
+```
 
 Razor pages expose a `Model` property for accessing the model passed to the page.
 
 ```html
```
Comment (scottaddie) [inline aspnetcore/mvc/views/razor.md]: Change to cshtml

File: aspnetcore/mvc/views/tag-helpers/intro.md [hunk 11]
```diff
@@ -67,8 +67,6 @@ To expose all of the Tag Helpers in this project (which creates an assembly name
 
 If your project contains an `EmailTagHelper` with the default namespace (`AuthoringTagHelpers.TagHelpers.EmailTagHelper`), you can provide the fully qualified name (FQN) of the Tag Helper:
 
-<!-- literal_block {"ids": [], "linenos": false, "xml:space": "preserve", "language": "html", "highlight_args": {"hl_lines": [3]}} -->
-
 ```html
```
Comment (scottaddie) [inline aspnetcore/mvc/views/tag-helpers/intro.md]: Change to cshtml

File: aspnetcore/mvc/views/working-with-forms.md [hunk 12]
```diff
@@ -329,7 +325,7 @@ The `Validation Message Tag Helper`  is used with the `asp-validation-for` attri
 
 ```HTML
```
Comment (scottaddie) [inline aspnetcore/mvc/views/working-with-forms.md]: Change to cshtml

**Comment by guardrex** [comment]
@scottaddie Done :+1:

**Comment by scottaddie** [comment]
@guardrex You rock! 🎸 Thanks for the quick turnaround on this one.

**Pull request merged**